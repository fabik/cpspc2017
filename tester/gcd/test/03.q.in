15
20 22 18 13 11 30 15 1 8 24 27 22 20 30 8
