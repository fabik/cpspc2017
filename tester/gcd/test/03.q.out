1
31
61
88
112
134
156
176
196
214
229
242
253
261
269
